setup.sh

```diff
@@ -10,13 +10,14 @@ commands=(
   "lando drush wunder_next:setup-user-and-consumer"
   "lando drush eshd -y"
   "lando drush eshs"
-  "lando npm i"
-  "lando npm run build"
-  "(lando npm run start&)"
   "lando drush en wunder_democontent -y"
+  "lando drush state:set wunder_democontent.disable_revalidation TRUE"
   "lando drush mim --group=demo_content --execute-dependencies"
   "lando drush pm-uninstall wunder_democontent migrate migrate_tools migrate_plus -y"
-  "lando npm-stop || true"
+  "lando drush state:del wunder_democontent.disable_revalidation"
+  "lando drush cron"
+  "lando npm i"
+  "lando npm run build"
```

Comment:
We can use here the same thing we do in silta, where we disable temporarily the revalidation. This way we can do things in a more natural order, and we don't need to start the frontend in the background etc.